tests/common/direct-link-query-test.js

```diff
@@ -11,14 +11,21 @@ describe('common.direct-link-query', () => {
   it('returns the ID if the URL contains a #annotations:<ID> fragment', () => {
```

Comment:
Might be good to clarify ID is the annotation id or even the direct-linked annotation id.